### DISCUSSION TITLE
Support for the Lapce dev team

### DISCUSSION THREAD

**Discussion by tirohia** [lapce/lapce#2519] [General]
Lapce has been on my watch list for a while now. For a pre-alpha version of anything, it's spectacular. 

That said, there's still a few things that are missing that make it not usable for me as a day to day environment. I would really, really like the dev team to fix a few things and add a few new features. As best I can tell though, the Lapce dev team are not getting paid to do this, which means that they are all either independently wealthy, or that development of Lapce will be happening when it can be fit in around everything else in life. 

VsCode/Codium annoys me. I'm not going anywhere near Jetbrains again. I've found Neovim to be a tool for people who want to build an IDE, not an IDE itself. Lapce by comparison, seems clean and fast, with a lot of potential. 

So. Is there a means for us to support the dev team? A Patreon or something? Given how excellent it looks at this stage of development, I'd happily put a little money in every month to help move Lapce along, if the devs wanted to devote more time to it. 

Cheers
Ben.

**Comment by panekj** [comment]
Please see https://github.com/lapce/lapce/issues/854
